tsconfig.json

```diff
@@ -1,103 +1,13 @@
 {
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig to read more about this file */
```

Comment:
dts-loader does not support comments on .json file, to I had to remove them.
I'll submit the bug to them and revert this once it gets fixed.